server/src/main/java/org/opensearch/discovery/InitializeExtensionsRequest.java

```diff
@@ -29,23 +29,32 @@ public class InitializeExtensionsRequest extends TransportRequest {
      * TODO change DiscoveryNode to Extension information
      */
     private final List<DiscoveryExtension> extensions;
+    private static final int OS_DEFAULT_PORT = 9200;
```

Comment:
(Preference) I prefer to make constants like this public, with a javadoc pointing to how one can override the default.